### ISSUE TITLE
Scattered data/files in eWaterCycle Jupyter machine

### ISSUE THREAD

**Issue by SarahAlidoost** [eWaterCycle/technicalPaperExampleNotebooks#21]
Some data/files of case1 and case 2 notebooks are located in different directories in eWaterCycle Jupyter machine. We need to move them to a shared/common location. The source of data and accessibility info should be added to the notebooks or to a readme in the repository. 

**Case1 notebook:** 

- rhine_05min_era5.ini, rhine_05min_era5_without_moselle.ini, no_moselle_landmask_05min_rhine.map in `/mnt/home/user42/technicalPaperExampleNotebooks/settingFiles`, and also in other users' home directories. 

  -  The latest version of these config files should be part of the `technicalPaperExampleNotebooks` repository.

- marrmot_ERA5_Moselle_2001_2016.mat in `/mnt/data/examples/technical_paper/marrmot/marrmotForcing/validation`
  - probably it should be moved to `/mnt/data/parameter-sets`

- Moselle.shp in `/mnt/data/examples/technical_paper/marrmot/marrmotForcing/Moselle.shp`
  - it should be pushed to the `technicalPaperExampleNotebooks` repository. 

**Case 2 notebook:**

- merrimack_05min_era5.ini in  `/mnt/data/examples/technical_paper/pcr-globwb`
  - it should be pushed to the `technicalPaperExampleNotebooks` repository. 
  
- pcrglob parametersets in `/mnt/data/examples/technical_paper/pcr-globwb/input`
  - it should be moved to `/mnt/data/parameter-sets`
 
- FLX_US-Ha1_FLUXNET2015_SUBSET_DD_1991-2012_1-4.csv in `/mnt/home/user42/technicalPaperExampleNotebooks` and also in other users' home directories. The source of the data is unknown.
   - can this be pushed to the `technicalPaperExampleNotebooks` repository?

**[Irrigation notebook](https://github.com/eWaterCycle/ewatercycle/blob/main/docs/examples/Irrigation.ipynb) in ewatercycle examples:**

- pcrglob parametersets in `/mnt/data/examples/technical_paper/pcr-globwb/input`
   - it should be moved to `/mnt/data/parameter-sets`
  
- pcrglobwb_merrimack.ini in` /mnt/home/user37/ewatercycle/docs/examples/pcrglobwb_merrimack.ini`
  - it should be pushed to `ewatercycle` repository. 

FYI @Peter9192 and @sverhoeven

**Comment by RolfHut** [comment]
solved with #31
